serving/docs/adapters.md

```diff
@@ -0,0 +1,180 @@
+# Adapters
+
+**Note that this API is experimental and is subject to change. Using it requires the environment variable feature flag `ENABLE_ADAPTERS_PREVIEW`.**
+
+DJL Serving has first class support for adapters.
+Adapters are patches or changes that can be made to a model to fine tune it for a particular usage.
+The benefit of adapters rather than whole model fine-tuning is that they are often smaller and easier to distribute alongside a base model.
+This can allow for multiple adapters used at the same system, and sometimes even in the same batch.
+
+With DJL Serving, it is possible to easily work with adapters.
+You can create models that accept adapters, use DJL Serving to manage your available adapters, and call models with adapters.
+
+For a concrete usage, see the [large model inference adapters example notebook](http://docs.djl.ai/docs/demos/aws/sagemaker/large-model-inference/sample-llm/multi_lora_adapter_inference.html).
```

Comment:
this link does not seem to work

Reply:
Yeah. It goes to the notebook from https://github.com/deepjavalibrary/djl-demo/pull/372. But, that was just merged and not built into our docs yet. When it is built in, it should work